frontend/src/components/Project/ProjectSidebar.vue

```diff
@@ -62,6 +62,7 @@ interface ProjectSidebarItem extends SidebarItem {
 
 const props = defineProps<{
   projectSlug?: string;
+  projectName?: string;
```

Comment:
Let's call it project id consistently. Project ID is the resource ID, where as the project name is the format of projects/project-id. And project title is the readable name.

Reply:
Updated